opennlp-tools/src/main/java/opennlp/tools/util/DownloadUtil.java

```diff
@@ -50,68 +45,81 @@ public enum ModelType {
     germanModels.put(ModelType.POS,
-        baseUrl + "models/ud-models-1.0/opennlp-de-ud-gsd-pos-1.0-1.9.3.bin");
+        BASE_URL + MODELS_UD_MODELS_1_0 + "opennlp-de-ud-gsd-pos-1.0-1.9.3.bin");
     germanModels.put(ModelType.TOKENIZER,
-        baseUrl + "models/ud-models-1.0/opennlp-de-ud-gsd-tokens-1.0-1.9.3.bin");
+        BASE_URL + MODELS_UD_MODELS_1_0 + "opennlp-de-ud-gsd-tokens-1.0-1.9.3.bin");
     available_models.put("de", germanModels);
 
     final Map<ModelType, String> englishModels = new HashMap<>();
     englishModels.put(ModelType.SENTENCE_DETECTOR,
-        baseUrl + "models/ud-models-1.0/opennlp-en-ud-ewt-sentence-1.0-1.9.3.bin");
+        BASE_URL + MODELS_UD_MODELS_1_0 + "opennlp-en-ud-ewt-sentence-1.0-1.9.3.bin");
     englishModels.put(ModelType.POS,
-        baseUrl + "models/ud-models-1.0/opennlp-en-ud-ewt-pos-1.0-1.9.3.bin");
+        BASE_URL + MODELS_UD_MODELS_1_0 + "opennlp-en-ud-ewt-pos-1.0-1.9.3.bin");
     englishModels.put(ModelType.TOKENIZER,
-        baseUrl + "models/ud-models-1.0/opennlp-en-ud-ewt-tokens-1.0-1.9.3.bin");
+        BASE_URL + MODELS_UD_MODELS_1_0 + "opennlp-en-ud-ewt-tokens-1.0-1.9.3.bin");
     available_models.put("en", englishModels);
 
     final Map<ModelType, String> italianModels = new HashMap<>();
     italianModels.put(ModelType.SENTENCE_DETECTOR,
-        baseUrl + "models/ud-models-1.0/opennlp-it-ud-vit-sentence-1.0-1.9.3.bin");
+        BASE_URL + MODELS_UD_MODELS_1_0 + "opennlp-it-ud-vit-sentence-1.0-1.9.3.bin");
     italianModels.put(ModelType.POS,
-        baseUrl + "models/ud-models-1.0/opennlp-it-ud-vit-pos-1.0-1.9.3.bin");
+        BASE_URL + MODELS_UD_MODELS_1_0 + "opennlp-it-ud-vit-pos-1.0-1.9.3.bin");
     italianModels.put(ModelType.TOKENIZER,
-        baseUrl + "models/ud-models-1.0/opennlp-it-ud-vit-sentence-1.0-1.9.3.bin");
+        BASE_URL + MODELS_UD_MODELS_1_0 + "opennlp-it-ud-vit-sentence-1.0-1.9.3.bin");
     available_models.put("it", italianModels);
 
     final Map<ModelType, String> dutchModels = new HashMap<>();
     dutchModels.put(ModelType.SENTENCE_DETECTOR,
-        baseUrl + "models/opennlp-nl-ud-alpino-sentence-1.0-1.9.3.bin");
+        BASE_URL + "models/opennlp-nl-ud-alpino-sentence-1.0-1.9.3.bin");
     dutchModels.put(ModelType.POS,
-        baseUrl + "models/ud-models-1.0/opennlp-nl-ud-alpino-pos-1.0-1.9.3.bin");
+        BASE_URL + MODELS_UD_MODELS_1_0 + "opennlp-nl-ud-alpino-pos-1.0-1.9.3.bin");
     dutchModels.put(ModelType.TOKENIZER,
-        baseUrl + "models/ud-models-1.0/opennlp-nl-ud-alpino-tokens-1.0-1.9.3.bin");
+        BASE_URL + MODELS_UD_MODELS_1_0 + "opennlp-nl-ud-alpino-tokens-1.0-1.9.3.bin");
     available_models.put("nl", dutchModels);
 
   }
 
-  public static BaseModel downloadModel(String language, ModelType modelType, Class<?> type)
-          throws IOException {
+  /**
+   * Triggers a download for the {@link DownloadUtil.ModelType}
```

Comment:
Missing period.

Reply:
Kind of proof that I'm a human worker. 😄

Will fix.